ovs/proto_trace.go

```diff
@@ -182,3 +204,39 @@ func checkForFinalFlow(s string) ([]string, bool) {
 
 	return matches, true
 }
+
+func checkForMegaFlow(s string) ([]string, bool) {
```

Comment:
I think this is a lot of code duplication. I'd like to refactor it to be less repetitive.

Reply:
easy to follow what's going on tho